pkg/cmd/local_run.go

```diff
@@ -0,0 +1,110 @@
+/*
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to You under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/spf13/cobra"
+)
+
+func newCmdLocalRun(rootCmdOptions *RootCmdOptions) (*cobra.Command, *localRunCmdOptions) {
+	options := localRunCmdOptions{
+		RootCmdOptions: rootCmdOptions,
+	}
+
+	cmd := cobra.Command{
+		Use:     "local-run [files to inspect]",
```

Comment:
Probably `kamel local run` would be better. So that `kamel local` can have other subcommands if needed in the future.

Reply:
There is a slight problem here, Cobra doesn't like me having the same name for sub-commands even if they are attached to different parent commands. `kamel run` and `kamel local run` seem to want to share the same flags. I'm getting this error:
`Error: flag accessed but not defined: source`

As soon as I rename `kamel local run` to something else, it works as expected.

Reply:
Added `local run` command.